From 4a6058b7e5fa97496d6548a985f714aaa9956410 Mon Sep 17 00:00:00 2001
From: Bian Jin chen <kenjc.bian@rock-chips.com>
Date: Wed, 4 Mar 2020 11:22:10 +0800
Subject: [PATCH] ARM: config: rockchip_defconfig: enable CONFIG_OVERLAY_FS.

Android 10 use overlayfs to update logical partitions,
enabling this config to support adb debugging.

Test: adb root; adb remount; adb push some_file /somewhere.

Signed-off-by: Bian Jin chen <kenjc.bian@rock-chips.com>
Change-Id: Ic6d90915a88208b82f771db203521e628a74fc2d
---
 arch/arm/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index e28f6e72595e..c3113fa2828a 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -639,6 +639,7 @@ CONFIG_QUOTA=y
 CONFIG_QUOTA_NETLINK_INTERFACE=y
 CONFIG_QFMT_V2=y
 CONFIG_FUSE_FS=y
+CONFIG_OVERLAY_FS=y
 CONFIG_ISO9660_FS=y
 CONFIG_JOLIET=y
 CONFIG_ZISOFS=y
-- 
2.35.3

